### ISSUE TITLE
Aqara curtain B1?

### ISSUE THREAD

**Issue by netRunner0** [ebaauw/homebridge-hue#462]
plugin supports?

**Comment by ebaauw** [comment]
Never heard of B1 before, but homebridge-hue does support the `lumi.curtain`.  If the B1 is supported by the deCONZ REST API, we’ll get it working with homebridge-hue.  Please list the resource(s) that the REST API provides.

**Comment by Larsn1** [comment]
Looks like parts from the postponed IKEA Fyrtur/Kadril blinds that were set to launch in February 2019.

**Comment by ebaauw** [comment]
Link?

**Comment by netRunner0** [comment]
https://ru.aliexpress.com/item/mi-Aqara-B1/32971304787.html

**Comment by netRunner0** [comment]
![IMG_0078](https://user-images.githubusercontent.com/44049295/56913446-226d3b00-6acb-11e9-802f-ac66d5e92dd6.jpeg)

**Comment by ebaauw** [comment]
That one looks like a battery-powered version of the `lumi.curtains`.  Nothing like the IKEA blinds.

Did you pair it with deCONZ?  Does it show in the REST API? If not, please open an issue with deCONZ, providing the info from https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Request-Device-Support.  If it does show in the REST API, homebridge-hue should pick it up   If not, please list the resource(s), or attach the debug dump file, see https://github.com/ebaauw/homebridge-hue#debug-dump-file.

**Comment by Larsn1** [comment]
If you look closely in the Ikea Video one could have the impression that this is the driver for the Ikea Smart blinds.

https://m.youtube./watch?v=S7Uqsjt7yQY

**Comment by ebaauw** [comment]
Link is broken?

**Comment by ebaauw** [comment]
Waiting on https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1483.  Closing this for now.

**Comment by netRunner0** [comment]
https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1654
